src/main/java/exceptions/DeadlineException.java

```diff
@@ -0,0 +1,25 @@
+package exceptions;
+
+/**
+ * A class that belongs to the Exceptions Package.
+ * This class encapsulates the message that should be displayed when users parsed an invalid input
+ * for the {@link tasks.DeadLines} class.
+ */
+public class DeadlineException extends TaskException {
```

Comment:
I like that you used inheritance to differentiate the different types of exceptions, instead of simply using different Strings in the respective exceptions' messages